game/game/models/character.py

```diff
@@ -119,7 +121,24 @@ async def remove_deleted_mark(self):
     def __hash__(self):
         return hash(f"{self.id}_{self.name}")
 
+    def update_shortcut(self, session, shortcut):
+        """
+        Update shortcut immediately
+        """
```

Comment:
Probably need to pin this docstrings convention in readme

```
def some_function(first_arg: int, second_arg: str, s*args, **kwargs):
    """Short function explanation in one line.

    Really long multiline function explanation 
    goes here if needed.

    :param first_arg: Some int value about info
    :param second_arg: Other param value info

    :raises ExceptionName: If raises and why
     
    :return What returns
```

Short version looks like this
```
def some_function(first_arg: int, second_arg: str, s*args, **kwargs):
    """Short function explanation in one line.""""
```